eng/tox/install_dev_build_dependency.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# Normally, this module will be executed as referenced as part of the devops build definitions.
+# An enterprising user can easily glance over this and leverage for their own purposes.
+import sys
+import argparse
+import logging
+from os import path
+from pip._internal.operations import freeze
+from subprocess import check_call
+
+# import common_task module
+root_dir = path.abspath(path.join(path.abspath(__file__), "..", "..", ".."))
+common_task_path = path.abspath(path.join(root_dir, "scripts", "devops_tasks"))
+sys.path.append(common_task_path)
+from common_tasks import process_glob_string
+from tox_helper_tasks import get_package_details
+
+EXCLUDED_PKGS = [
+    "azure-common",
+]
+
+logging.getLogger().setLevel(logging.INFO)
+
+# This script verifies installed package version and ensure all installed pacakges are dev build version
+
+
+def get_installed_packages(pkg_name_to_exclude):
+    # This method returns a list of installed azure sdk packages
+    installed_pkgs = [
+        p.split("==")[0] for p in freeze.freeze() if p.startswith("azure-")
+    ]
+
+    # Get valid list of Azure SDK packages in repo
+    pkgs = process_glob_string("", root_dir)
+    valid_azure_packages = [
+        path.basename(p) for p in pkgs if "mgmt" not in p and "-nspkg" not in p
```

Comment:
Note that this won't catch `cognitive-services`, although I think that's a good thing.

Reply:
Yeah, this is a good thing, given that some cogservices are track 2 now. Leave this alone.